lib/util.js

```diff
@@ -199,6 +199,13 @@ function deprecate(text) {
   /* eslint-enable */
 }
 
+function deprecateWrap(text, fn, that) {
```

Comment:
And am wondering here, if we don't care about "this", we don't need to pass the context arround (can just run an apply(null, arguments)